.github/workflows/publish_on_release.yml

```diff
@@ -0,0 +1,41 @@
+# Publish releases to Clojars
+on:
+  release:
+    types: [published]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      # SETUP START
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Prepare java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: '8'
+
+      - name: Install clojure tools
+        uses: DeLaGuardo/setup-clojure@10.2
+        with:
+          cli: 1.11.1.1200            # Clojure CLI based on tools.deps
+
+      - name: Cache clojure dependencies
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.m2/repository
+            ~/.gitlibs
+            ~/.deps.clj
+          # List all files containing dependencies:
+          key: cljdeps-${{ hashFiles('deps.edn') }}
+          restore-keys: cljdeps-
+      # SETUP END
+
+      - name: Build pipeline
+        run: >-
+          clojure -X:build pipeline :lib io.github.mortenschioler/foo :version '"${{github.ref_name}}"'
```

Comment:
did you intend for this to be `foo`?

Reply:
No, good eye! I realized that I forgot to change this on my bike trip home, and thought I'd change it tomorrow, but you were too quick for that! 😁

Reply:
Fixed in 5b013a9.